GANDLF/metrics/segmentation.py

```diff
@@ -64,37 +71,44 @@ def multi_class_dice(output, label, params, per_label=False):
         return total_dice
 
 
-def multi_class_dice_per_label(output, label, params):
+def multi_class_dice_per_label(
+    prediction: torch.Tensor, target: torch.Tensor, params: dict
+) -> list:
     """
     This function computes a multi-class dice.
 
     Args:
-        output (torch.Tensor): Input data containing objects. Can be any type but will be converted into binary: background where 0, object everywhere else.
-        label (torch.Tensor): Input data containing objects. Can be any type but will be converted into binary: background where 0, object everywhere else.
+        prediction (torch.Tensor): Input data containing objects. Can be any type but will be converted into binary: background where 0, object everywhere else.
+        target (torch.Tensor): Input data containing objects. Can be any type but will be converted into binary: background where 0, object everywhere else.
         params (dict): The parameter dictionary containing training and data information.
 
     Returns:
         list: The list of per-label dice scores.
     """
-    return multi_class_dice(output, label, params, per_label=True)
+    return multi_class_dice(prediction, target, params, per_label=True)
 
 
-def __surface_distances(result, reference, voxelspacing=None, connectivity=1):
+def __surface_distances(
+    prediction: torch.Tensor,
+    target: torch.Tensor,
+    voxelspacing: tuple = None,
```

Comment:
Tuple type shoud be added and also the argument should be of Optional type

Reply:
Fixed.